Mdh bridge rework

ma-schulze opened the pull request:
- uncommented running docker as a user, as this still wasnt working that well
- removed the open source metadatahub from the init und run.sh
- Reworked the mdh-fuse bridge to now work with the new mdh
ATTENTION!!!
  - Of all the new commands/queries not everything works, a lot is still broken. Only the very basic functionality needed for the traversing is somewhat tested and works so far. EVERYTHING ELSE IS WIP, so use with caution 
  - I would still really like to merge this, as this basic functionality will be needed for others for their work

sandraaocode committed:
Add files via upload

sandraaocode committed:
Add files via upload

sandraaocode committed:
Delete Kanban Board_Sprint 3.png

tuhinmallick committed:
Wrong name of the planning document

This is the planning document for the 3rd sprint and not 2nd

tuhinmallick committed:
Correct Planning Document uploaded

tuhinmallick committed:
Screenshot of Sprint 3 uploaded

vaidehi-for-coding committed:
Merge pull request #28 from amosproj/Dev

Dev

ma-schulze committed:
- uncommented running docker as a user, as this still wasnt working that well
- removed the open source metadatahub from the init und run.sh
- Reworked the mdh-fuse bridge to now work with the new mdh
ATTENTION!!!
  - Of all the new commands/queries not everything works, a lot is still broken. Only the very basic functionality needed for the traversing is somewhat tested and works so far. EVERYTHING ELSE IS WIP, so use with caution

MarlonWe commented on src/mdh_bridge.py:
```diff
@@ -0,0 +1,363 @@
+    query_root.queries.append(query)
+    query_root.build_and_send_request()  # type: MetadataResult
+    metadatahub_files = query_root.queries[0].result.files
+    import fuse_utils
```
why is the import here in function scope?

MarlonWe commented on src/mdh_bridge.py:
```diff
@@ -0,0 +1,363 @@
+    print(RenderTree(directory_tree))
+
+
+    rq = MDHQueryRoot()
```
i guess the variable names are short because of speed reasons :D

Commit:
Fixed linting errors

Commit:
Removed complexity check

ma-schulze committed:
- added some comment regarding the "testcase" in the mdh_bridge
- small changes to how the DirectoryTree in the fuse_utils.py is created
- small changes to the stats struct

ma-schulze committed:
- fixed duplicate class
- some cleanup
